crypto/cipher/aes_icm.c

```diff
@@ -199,8 +199,8 @@ static srtp_err_status_t srtp_aes_icm_context_init (void *cv, const uint8_t *key
 
     copy_len = c->key_size - base_key_len;
     /* force last two octets of the offset to be left zero (for srtp compatibility) */
-    if (copy_len > 14) {
-        copy_len = 14;
+    if (copy_len > SRTP_SALT_LEN) {
```

Comment:
This should already have been verified I guess so this check should be redundant, it was not my intention to change the meaning of the code only the use of a define. If you find that the code was wrong then that should be changed in a separate PR.